client/src/context/UserContext.js

```diff
@@ -0,0 +1,11 @@
+import React, { createContext, useContext } from "react";
+import { useAuth } from "./AuthContext";
+
+const UserContext = createContext();
+const UserProvider = (props) => (
+  <UserContext.Provider value={useAuth().data.user} {...props} />
```

Comment:
Do you need a separate context as it just provides a value that could already provided by AuthContext?

Maybe you could add this function to your AuthContext.js:
```
export function useUser() {
  return useAuth().data.user;
}
```